client/src/components/Section/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './main.scss';
+import { AdsBlock } from '../AdsBlock/index';
+import { SaleItem } from '../SaleItem/index';
+
+export class Section extends React.Component {
+  render() {
+    return (
+      <div className="section">
```

Comment:
Class name section is too abstract, I think you can change it to, f. e. , stock and change this div class='section' to html5 section tag